examples/kubernetes/quickstart.yaml

```diff
@@ -15,6 +15,15 @@ spec:
     - name: http
       protocol: HTTP
       port: 8080
+    - name: https
+      protocol: HTTPS
+      port: 8443
+      tls:
+        mode: Terminate
+        certificateRefs:
+          - kind: Secret
+            group: ""
+            name: example-cert
```

Comment:
Just noting here that users who are just using the regular quickstart will see this Listener marked as invalid in their Gateway conditions since we're not creating a TLS cert in the quickstart so the ref is invalid -- HTTP routing still works fine, just may be confusing. You could use a separate example YAML for the secure gateway guide, or add a kubectl command to add this listener in the secure gateway guide, or leaving as-is is probably fine too for now.

Reply:
I removed this listener from the quickstart manifest and update the example gateway in this doc.